presto-main/src/main/java/com/facebook/presto/sql/planner/plan/MarkDistinctNode.java

```diff
@@ -89,6 +95,12 @@ public Symbol getMarkerSymbol()
         return hashSymbol;
     }
 
+    @JsonProperty
+    public List<Symbol> getOrderBySymbols()
```

Comment:
I don't see usage of this method and so it looks like `orderBySymbols` can be removed from this class.

The only usage is here: com.facebook.presto.sql.planner.iterative.rule.SingleMarkDistinctToGroupBy, which seems that it can be changed